lib/cudadrv/stream.jl

```diff
@@ -98,6 +99,19 @@ function query(s::CuStream)
     end
 end
 
+"""
+    synchronize([s::CuStream]; blocking=true)
+
+Wait until a stream's tasks are completed. If `blocking` is true (the default), Julia will
+be asked to yield to any other scheduled task.
+"""
+function synchronize(s::CuStream=stream(); blocking::Bool=true)
```

Comment:
`blocking` is a bit of an odd name. The function is blocking nonetheless.

Reply:
You're right; It's because of cudaEventBlockingSync that I chose this name: https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__TYPES.html#group__CUDART__TYPES_1g26509a522be9d449aa7c8c279612452d
> cudaEventBlockingSync: Specifies that event should use blocking synchronization. A host thread that uses cudaEventSynchronize() to wait on an event created with this flag will block until the event actually completes.

Reply:
Let's keep it `blocking` to mirror CUDA, unless you feel strongly about this?